### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by Kaghenimbale**
## For this Personal Portfolio project, I have done the following steps:
* [ ]  I add the works section
* [ ]  Modify the about me section
* [ ]   Add the ul li list item in the about me
* [ ]  Use CSS grid and flexbox

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
```
Comment (christianbiring1) [inline README.md]: - [ ] Kindly make sure you have provided the command to clone the repos, and navigate through it ...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
```
Comment (christianbiring1) [inline README.md]: - [ ] Kindly make sure you provide the commands.
`Hint: YOu may say`
```sh
cd <directory_name>
Open the index.html in the browser
...
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+```sh
+  cd my-project
+  open the file index.html in your desired browser
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
```
Comment (christianbiring1) [inline README.md]: - [ ]   YOu may consider providing the command to checks the linters errors.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,411 @@
+/* stylelint-disable CssSyntaxError */
+/* stylelint-disable no-descending-specificity */
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure you stick to the design as much as you can. the typo and the spacing are supposed to match the Figma design. make sure you fix that.
Hint: In Figma they use `Roboto` you will need to use the `google fonts service`
You may check [here](https://www.youtube.com/watch?v=qgmLDPLApBY) how to do so

File: style.css [hunk 5]
```diff
@@ -0,0 +1,411 @@
...
+  color: #172b4d;
+  padding: 0 20px;
+  line-height: 1.4;
+}
+
+.section-1 p {
+  font-size: 1.2rem;
+  color: #344563;
+  padding: 0 20px;
+  line-height: 1.5;
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure you do not have any kind of scroll in any of the sections on your page as you may notice how all of them are. `The introduction part, the let's connect text.` 
`Hint: You may consider removing the height:80vh applied on this section and the different line-height.`
Your design :point_down: 
image
Figma template :point_down: 
image

File: style.css [hunk 6]
```diff
@@ -0,0 +1,411 @@
...
+  padding: 0 20px;
+  align-items: center;
+  gap: 15px;
+  height: 50px;
+}
+
+.section-1 .nav-links ul li {
+  width: 30px;
+  height: 30px;
+}
```
Comment (christianbiring1) [inline style.css]: - [ ] As part of what I have said above. Kindly consider removing the individual `heigh` applied to these #ts elements

File: style.css [hunk 7]
```diff
@@ -0,0 +1,411 @@
...
+  height: 30vh;
+  overflow: hidden;
+}
+
+.container .profile img {
+  width: 100%;
+  height: 100%;
+  border: 10px;
+  border-radius: 15px;
+}
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure you stick to the image as much as possible. the spacing, typo are supposed to match the figma design.
`Hint: You may consider making your image responsive by applying the:`
```css
width: 100%,
height: auto
// also you consider setting the
object-fit: container
```
- [ ] Kindly make sure the list element of `backend ...` are well aligned as in the template
Your design :point_down: 
image

The Figma template :point_down: 
image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+                    <div class="paragraph-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="languages">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure you have the correct fonts (family and size on your element)
Your design :point_down: 
image
Figma template :point_down: 
image

File: index.html [hunk 9]
```diff
@@ -0,0 +1,191 @@
...
+            <h2>About <br> Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="connect">LET'S CONNECT</p>
+            <nav class="nav-links">
+                <ul>
+                    <li><a href="#"><img src="image/github.svg" alt="twitter"></a></li>
+                    <li><a href="#"><img src="image/Linkedin icon.svg" alt="Linkedin"></a></li>
+                    <li><a href="#"><img src="image/Vector-1.svg" alt="Vector-1"></a></li>
+                    <li><a href="#"><img src="image/twitter.svg" alt="github"></a></li>
+                    <li><a href="#"><img src="image/Vector.svg" alt="Vector"></a></li>
```
Comment (christianbiring1) [inline index.html]: [OPTIONAL]
- [ ] It would be great to link these links to your social handlers.

File: style.css [hunk 10]
```diff
@@ -0,0 +1,411 @@
...
+  border-radius: 5px;
+  border: 1px solid #396df2;
+  margin-bottom: 10px;
+}
+
+button a {
+  font-size: 1.1rem;
+  color: #396df2;
+}
+
```
Comment (christianbiring1) [inline style.css]: - [ ] As is the case on the template, kindly make sure the buttons on your page follow the button styling: Active, Hover, and Pressed (see design)

image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Kaghenimbale , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR`title` and `summary`
- [ ] The flexbox is used correctly to place the items on the page :art: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bienvenuushindi — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @Kaghenimbale   👋, 

Well done implementing the requested changes 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights 🎯 
:+1: No linter errors✔️
:+1: Good UI design✔️
:+1: Good PR title

### Optional Suggestions



**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**